client/js/map.js

```diff
@@ -16,11 +16,11 @@
 geoapp.TileSets = {
     mapquest: {
         url: 'http://otile1.mqcdn.com/tiles/1.0.0/map/',
-        credit: 'Tiles Courtesy of <a href="http://www.mapquest.com/">MapQuest</a>'
+        credit: 'Tiles courtesy of <a href="http://www.mapquest.com/">MapQuest</a>'
```

Comment:
It probably doesn't matter, but I copied the capitalization from MapQuest's developer's page: http://developer.mapquest.com/web/products/open/map.